spec/migrations/20181001131632_add_conversion_host_id_to_miq_request_tasks_spec.rb

```diff
@@ -0,0 +1,56 @@
+require_migration
+
+describe AddConversionHostIdToMiqRequestTasks do
+  let(:task_stub) { migration_stub(:MiqRequestTask) }
+  let(:host_stub) { migration_stub(:Host) }
+  let(:conversion_host_stub) { migration_stub(:ConversionHost) }
+  let(:host_id) { 42 }
+
+  migration_context :up do
+    it "when host doesn't exist" do
+      task = task_stub.create!(
+        :type    => 'ServiceTemplateTransformationPlanTask',
+        :options => { :dummy_key => 'dummy_value', :transformation_host_id => host_id }
+      )
+
+      migrate
+      task.reload
+
+      expect(task.options).to eq({ :dummy_key => 'dummy_value', :transformation_host_id => host_id })
```

Comment:
Hm is this right?  If it can't find the host it should still clear the transformation_host_id right?

Reply:
I think this is what I was talking about in https://github.com/ManageIQ/manageiq-schema/pull/281/files#r222028373 @agrare

Reply:
Ha yeah I was in the middle of linking this to your comment above